source/_integrations/prometheus.markdown

```diff
@@ -204,3 +204,9 @@ For example:
 - record: "known_temperature_c"
   expr: "temperature_c unless entity_available == 0"
 ```
+
+## Supported metrics
+
+Metrics are exported only for the following domains:
+
+`automation`, `binary_sensor`, `climate`, `cover`, `counter`, `device_tracker`, `fan`, `humidifier`, `input_boolean`, `input_number`, `light`, `lock`, `number`, `person`, `sensor`, `update`, `switch`, `zwave`
```

Comment:
This certainly helps set expectations, but I worry it will become a maintenance burden. Now every time someone adds support for another domain, they will need to update the documentation page.

Also, support for a particular domain is not binary: for example, we already had metrics for the `climate` domain, but [your recent PR](https://github.com/home-assistant/core/pull/120267/) adds more (thank you!).

I wonder if there is a way to communicate the fact that `prometheus` integration does not provide automatic instrumentation of all entities and state attributes without hard-coding a list of supported domains here.

Reply:
I agree it's too much of a maintenance burden.  I would prefer the following note instead, but @frenck asked me to remove the link to code.

> Note: Metrics are exported only for certain domains. For details, see the `def _handle_` functions in the [Prometheus integration source code](https://github.com/home-assistant/core/blob/dev/homeassistant/components/prometheus/__init__.py).